src/cmd/build.js

```diff
@@ -104,6 +106,51 @@ export async function getDefaultLocalizedName(
   return Promise.resolve(extensionName);
 }
 
+// https://stackoverflow.com/a/22129960
+function getStringPropertyValue(
```

Comment:
Nit, we could export this function and unit test a couple of the other cases by calling it directly (e.g. to quickly test its behavior on a number property set to `0`, or an empty string etc.).